app/assets/stylesheets/govuk_publishing_components/components/_layout-header.scss

```diff
@@ -10,3 +10,22 @@
 .gem-c-layout-header--test .govuk-header__container {
   border-bottom-color: govuk-colour("grey-1");
 }
+
+.govuk-header__logo {
+  white-space: nowrap;
+}
+
+.gem-c-environment-tag {
```

Comment:
After this sits inside the anchor, can we have:
```
padding: 2px 5px 0;
vertical-align: middle;
@include govuk-font($size: 14);
```
This will make it consistent with the other GDS products